### PR TITLE
Adds support for Fyrtur and Kadrilj roller blinds

### PR DESCRIPTION
**Pull request by RobinBol**
Supported functionality:
- Moving blinds up and down to a relative percentage using a dim slider
- Displaying current battery levels

Todo's
- [X] Add developer contribution
- [x] Device icons
- [x] Learnmode icons

### CODE REVIEW

**Comment by TvAnton** [comment]
Hi how do download this uppdate?

**Comment by RobinBol** [comment]
> Hi how do download this uppdate?

Just keep an eye on the beta channel of this app in the Homey Apps store!

File: app.json [hunk 1]
```diff
@@ -893,6 +901,80 @@
           }
         }
       ]
+    },
+    {
+      "id": "rollerblind_fyrtur",
+      "name": {
+        "en": "Fyrtur Rollerblind",
+        "nl": "Fyrtur Rolgordijn",
+        "de": "Furtur Rollo"
```
Comment (KyleAthom) [inline app.json]: Typo in productname

File: app.json [hunk 2]
```diff
@@ -893,6 +901,80 @@
...
+          "FYRTUR block-out roller blind"
+        ],
+        "deviceId": 514,
+        "profileId": 260,
+        "learnmode": {
+          "image": "/drivers/rollerblind_fyrtur/assets/learnmode.svg",
+          "instruction": {
+            "en": "Simultaneously press the up and down buttons on the rollerblind until the LED starts blinking.\n\nNote: it can take 15-20 seconds before pairing starts, if the device does not pair within 15-20 seconds please try again.",
+            "nl": "Druk tegelijk de twee omhoog en omlaag knoppen op het rolgordijn in totdat de LED begint te knipperen.\n\nLet op: het kan 15-20 seconden duren voor het apparaat wordt gekoppeld, gebeurt dit niet binnen 15-20 seconden probeer het dan nog eens."
+          }
```
Comment (KyleAthom) [inline app.json]: Missing DE translation?

File: app.json [hunk 3]
```diff
@@ -893,6 +901,80 @@
...
+          "KADRILJ roller blind"
+        ],
+        "deviceId": 514,
+        "profileId": 260,
+        "learnmode": {
+          "image": "/drivers/rollerblind_kadrilj/assets/learnmode.svg",
+          "instruction": {
+            "en": "Simultaneously press the up and down buttons on the rollerblind until the LED starts blinking.\n\nNote: it can take 15-20 seconds before pairing starts, if the device does not pair within 15-20 seconds please try again.",
+            "nl": "Druk tegelijk de twee omhoog en omlaag knoppen op het rolgordijn in totdat de LED begint te knipperen.\n\nLet op: het kan 15-20 seconden duren voor het apparaat wordt gekoppeld, gebeurt dit niet binnen 15-20 seconden probeer het dan nog eens."
+          }
```
Comment (KyleAthom) [inline app.json]: Missing DE translation?

**Pull request closed without merging**